django/template/loader_tags.py

```diff
@@ -207,6 +210,8 @@ def render(self, context):
         except Exception:
             if context.template.engine.debug:
                 raise
+            else:
+                logger.warn('template - Uncaught Exception raised during rendering.', exc_info=True)
```

Comment:
I'm not sure if the included traceback will give enough information to debug this, but it seems like a message something like "Rendering `<template name>` raised an exception, so {% include %} will render as an empty string." might be more helpful.